core/templates/components/summary-tile/topic-summary-tile.directive.html

```diff
@@ -10,9 +10,10 @@
     </div>
     <div class="topic-details" ng-style="{'background-color': $ctrl.getDarkerThumbnailBgColor()}" ng-class="{'mask-effect': !$ctrl.isPublished()}">
       <span class="topic-name"> <[$ctrl.getTopicSummary().getName()]> </span>
-      <span class="topic-story-count" ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() > 0">
-        <[$ctrl.getTopicSummary().getCanonicalStoryCount()]>
-        <span ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() === 1" translate="I18N_TOPIC_VIEWER_STORY"></span>
+      <span class="topic-story-count" ng-if="$ctrl.getTopicSummary().getTotalPublishedChapterCount() > 0">
+        <[$ctrl.getTopicSummary().getTotalPublishedChapterCount()]>
+        <!-- Add `I18N_TOPIC_VIEWER_CHAPTERS` -->
+        <span translate="I18N_TOPIC_VIEWER_CHAPTER"></span>
```

Comment:
How are we handling plurals here?

Reply:
Done. Tested by manually setting `nodeCount=1` and `nodeCount=2`.